src/FakeItEasy/Core/FakeManager.PropertySetterRule.cs

```diff
@@ -10,7 +10,12 @@ public partial class FakeManager
         private class PropertySetterRule
             : IFakeObjectCallRule
         {
-            public FakeManager FakeManager { get; set; }
+            private readonly FakeManager fakeManager;

```

Comment:
Yes, I think in this case it was public readable by accident, since the getter was never used outside the class. The class was written in the same pattern as the other Rule classes.